src/api/api.js

```diff
@@ -11,6 +11,10 @@ class Api {
     this.validator = new Validator();
   }
 
+  static isSingleEmail(email) {
+    return !/[,\s<>]/.test(email);
```

Comment:
There appear to be a few modules out there that validate email address syntax according to the RFC.  Would it be more appropriate to sanity check that the address syntax is truly valid before we attempt to send an email?

Reply:
Found this site: https://emailregex.com
Will use the proposed JavaScript regex

Reply:
I should have clarified my concern better.  I was thinking of the non-malicious case where a user simply entered a malformed email address and had to wait for us to actually try to send the email before receiving an error message indicating the address is invalid.  I forgot to check the client side, and now see that you're using an `email` input element on the form.  That should handle client validation sufficiently to address the scenario I had in my mind. :+1:

But a better regex is still an improvement for our (future) REST API users. :smile: